README.md

```diff
@@ -35,7 +30,7 @@ but if you're overriding methods then please test your work before upgrading.
 
 ## Reporting Issues
 
-Please [create an issue](https://github.com/silverstripe/silverstripe-ckan-registry/issues)
+Please [create an issue](https://github.com/silverstripe/silverstripe-security-extensions/issues/new)
```

Comment:
These changes are all done in #1, no need to double up

Reply:
undone in #2